src/workspaces/features/WorkspaceDescriptionDialog/WorkspaceDescriptionDialog.tsx

```diff
@@ -0,0 +1,91 @@
+import Button from "core/components/Button";
+import Dialog from "core/components/Dialog";
+import Spinner from "core/components/Spinner";
+import useForm from "core/hooks/useForm";
+import { useEffect } from "react";
+import { useTranslation } from "react-i18next";
+import { Workspace } from "graphql-types";
+import Textarea from "core/components/forms/Textarea";
+import { useUpdateWorkspaceMutation } from "workspaces/graphql/mutations.generated";
+import useCacheKey from "core/hooks/useCacheKey";
+
+type WorkspaceDescriptionDialogProps = {
+  onClose(): void;
+  open: boolean;
+  workspace: Omit<
+    Workspace,
+    "createdAt" | "updatedAt" | "createdBy" | "memberships" | "countries"
+  >;
+};
+
+type Form = {
+  description: string;
+};
+
+const WorkspaceDescriptionDialog = (props: WorkspaceDescriptionDialogProps) => {
+  const { t } = useTranslation();
+  const { open, onClose, workspace } = props;
+  const clearCache = useCacheKey(["workspace", workspace.id]);
```

Comment:
This is not needed here since you can get the description from the return of the mutation

Reply:
Got it.